Core/clim-core/dialogs/menu-choose.lisp

```diff
@@ -111,22 +110,26 @@
                   (ecase (menu-item-option item :type :item)
                     (:item
                      ;; This is a normal item, just output.
-                     (let ((activep (menu-item-option item :active t)))
-                       (with-presentation-type-decoded (name params options)
-                           presentation-type
-                         (let ((*allow-sensitive-inferiors* activep))
-                           (with-text-style
-                               (stream (menu-item-option
-                                        item :style
-                                        '(:sans-serif nil nil)))
-                             (with-output-as-presentation
-                                 (stream
-                                  item
-                                  `((,name ,@params)
-                                    :description ,(getf (menu-item-options item) :documentation)
-                                    ,@options)
-                                  :single-box t)
-                               (funcall item-printer item stream)))))))
+                     (push
+                      (let ((activep (menu-item-option item :active t)))
+                        (with-presentation-type-decoded (name params options)
+                            presentation-type
+                          (let ((*allow-sensitive-inferiors* activep))
+                            (with-text-style
+                                (stream (menu-item-option
+                                         item :style
+                                         '(:sans-serif nil nil)))
+                              (with-output-as-presentation
+                                  (stream
+                                   item
+                                   `((,name ,@params)
+                                     :description ,(getf (menu-item-options item) :documentation)
+                                     ,@options)
+                                   :single-box t)
+                                (funcall item-printer item stream))))))
```

Comment:
I'm not sure whether draw-standard-menu should collect presentations, perhaps it should be frame-manager-menu-choose by making a closure over item-printer (and then estabilish the context with accelerators around the call to menu-choose-from-drawer).